modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/paths/PRMMDFRequest.path

```diff
@@ -110,6 +110,16 @@
 	<td>//div[contains(@class,'edit-button-mdf')]/div</td>
 	<td></td>
 </tr>
+<tr>
+	<td>REMOVE_VALUE_BUDGET_BUTTON</td>
+	<td>//button[contains(@class,'btn-monospaced')]/*[name()='svg'][contains(@class,'lexicon-icon lexicon-icon-hr')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>VALUE_BUDGET</td>
+	<td>//div[contains(@class,'overflow-auto') and contains(.,'key_fieldName') and contains(.,'key_fieldValue')]</td>
+	<td></td>
+</tr>
```

Comment:
Hi @Italo-gabriel, you forgot to put your variables between `${}`.

Reply:
Also, I think that this path will be better like this: `//*[@route='mdf-request-form']//div[contains(@class,'text-paragraph') and contains(text(),'Total cost')]//following-sibling::div[contains(@class,'text-paragraph') and contains(text(),'${key_totalCost}')]`. What do you think?

Reply:
make sense , I changed the path , thanks !